requirements.txt

```diff
@@ -4,13 +4,18 @@ async-timeout==3.0.0
 attrs==18.1.0
 autobahn==18.6.1
 Automat==0.7.0
+certifi==2018.4.16
 channels==2.1.2
 channels-redis==2.2.1
+chardet==3.0.4
 constantly==15.1.0
 coverage==4.5.1
+coveralls==1.3.0
```

Comment:
It looks like in other projects, because coveralls is something we only use with Travis, we install it as part of the Travis set up - https://github.com/praekelt/molo/blob/develop/.travis.yml#L55